package.json

```diff
@@ -22,6 +22,7 @@
         "eslint-plugin-mocha": "~8.1.0",
         "husky": "^6.0.0",
         "jsonschema": "~1.4.0",
+        "markdownlint-cli": "^0.28.1",
```

Comment:
Where do we use it? Will there be a job for it or do you intend to use it manually?

Reply:
markdownlint script uses this dependency

Reply:
I see this but where do we use that script?

Reply:
I put it inside "lint" and "lint:fix" scripts.

Reply:
Ah, OK.